### PR TITLE
Michael/dashboard issue fix 992

### PR DESCRIPTION
**Pull request by MichaScant**
# 🔗 Issue
https://github.com/rubyforgood/pet-rescue/issues/992

# ✍️ Description
I have edited the relevant foster and adoption application files under staff to have the same HTML classes which should solve the sizing issue. Below you are able to see the pages layered on top of each other as a demo.

# 📷 Screenshots/Demos
table_comparison (1)

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -56,6 +56,15 @@
     t.index ["pet_id"], name: "index_adopter_applications_on_pet_id"
   end
 
+  create_table "adopter_foster_accounts", force: :cascade do |t|
```
Comment (kasugaijin) [inline db/schema.rb:59]: This table should not be committed as we deleted it recently. Please remove this (might want to rebase with main branch)

**Comment by kasugaijin** [comment]
Thank you! Could you please provide a video or screenshots of the UI changes? Helps with review a lot!

**Comment by MichaScant** [comment]
Hi! Thank you for replying, the test cases should now be passing, here are some more screenshots comparing the before and after shots of the UI changes I made. 
Before :
Screenshot 2024-09-25 at 11 38 08 AM
Screenshot 2024-09-25 at 11 37 58 AM

After:
Screenshot 2024-09-25 at 11 40 13 AM
Screenshot 2024-09-25 at 11 40 07 AM

**Review by kasugaijin — APPROVED** [review]
LGTM! Thank you @MichaScant

**Pull request merged**